.github/workflows/tests-all.yml

```diff
@@ -0,0 +1,37 @@
+name: Smoke Test
```

Comment:
We already have a workflow with this name. This might cause issues in GHA.

Reply:
```suggestion
name: Test All
```